### ISSUE TITLE
Spectra from Thermo mzML files is hard to read

### ISSUE THREAD

**Issue by benbowen** [biorack/BASTet#1]
@oruebel Yesterday we uploaded a file that was acquired on a MALDI LTQ/XL (Thermo) and had been converted to mzML using msconvert with default parameters.  BASTet correctly identified the unique m/z values and binned the data on m/z axis, but its really hard to read when plotted as a line plot.
![image](https://cloud.githubusercontent.com/assets/810953/15860354/8d018c4e-2c7d-11e6-8302-8b6082c3f8a3.png)
I think adding ~5ppm binning as a minimum failsafe would help prevent this.  

Basically debunch the closely spaced m/z values and then add in m/z values where closest is far away.

I'd be happy if this was in "analysis_0" so the raw data could be preserved.  With the new jupyter.nersc.gov setup users should be able to add custom analysis to their openmsi files without much trouble at all.

**Comment by oruebel** [comment]
Can you send me the raw file so that I can have a look at the convert process. If the data is profiled, then the default rebinning should use 5ppm already but depending on whether the file is Bruker or Thermo there are some different paths the code is taking. The functions in the omsi/dataformat/mzml_file.py reader to look at are 1) __compute_mz_axis(..) which computes the mz axis and 2) spectrum_iter(..) which retrieves (and possible reinterpolates). We should make sure that the functions do what you expect.

**Comment by benbowen** [comment]
The file above is from: [this file I just shared with you](https://openmsi.nersc.gov/openmsi/openmsi/client/viewer/?file=%2Fproject%2Fprojectdirs%2Fopenmsi%2Fomsi_data_private%2Fbpb%2F20160526_MDR_Orbi_Maldi_ARGO_cel_scr1.h5&dataIndex=0&expIndex=0&channel1Value=1000&channel1RangeValue=0.0637215419594&channel2Value=1250&channel2RangeValue=0.0637215419594&channel3Value=1500&channel3RangeValue=0.0637215419594&rangeValue=0.0637215419594&cursorCol1=53&cursorRow1=50&cursorCol2=107&cursorRow2=100&enableClientCache=false)

The other Orbitrap files markus has uploaded are here:
bpb@cori02:~> ls /project/projectdirs/openmsi/original_data/raad0102/ | grep Orbi
20160406_MDR_MaldiOrbi_Maldi_ARGO_1
20160428_MDR_MOrbi_Maldi_ARGO_quant_1
20160525_MDR_Orbi_Maldi_ARGO_cello_t2
20160526_MDR_Orbi_Maldi_ARGO_cel_scr1
20160526_MDR_Orbi_Maldi_LCMS_samples
20160531MdR_MaldiOrbi_Maldi_ARGO_AT_reactions_3_Sigma_AT
20160601_MDR_Orbi_Maldi_ARGO_GH_scr2
